folio-spring-cql/src/main/java/org/folio/spring/cql/Cql2JpaCriteria.java

```diff
@@ -365,11 +370,15 @@ private String getFieldNameByModifier(Path<?> entity, Modifier modifier) {
    * Create an SQL expression using LIKE query syntax.
    */
   private static Predicate queryByLike(Path<String> field, String term, String comparator,
-                                CriteriaBuilder cb) {
+                                CriteriaBuilder cb, boolean ignoreCase) {
     if (NOT_EQUALS_OPERATOR.equals(comparator)) {
-      return cb.notLike(field, cql2like(term), '\\');
+      return ignoreCase
+        ? cb.notLike(cb.lower(field), cql2like(term).toLowerCase())
```

Comment:
```suggestion
        ? cb.notLike(cb.lower(field), cql2like(term).toLowerCase(), '\\')
```
Shouldn't like ignore case queries also have escape char?

Reply:
done